vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/ModuleWithSpace.java

```diff
@@ -0,0 +1,15 @@
+package edu.asu.diging.vspace.core.model.impl;
```

Comment:
this package contains our entities that are stored in the db, but this one is not if I'm not mistaken? So it shouldn't be in this package.